about.html

```diff
@@ -22,24 +21,19 @@
 <h3>Applications</h3>
 
 <ul>
-<li>PcManFm-qt - File Manager</li>
-<li>Lximage-qt - Image Viewer</li>
-<li>QTerminal - Terminal Emulator</li>
-<li>Qps - Process Viewer</li>
-<li>Screengrab - Screen Capturer</li>
-<li>LXQt-archiver - Archive Manager</li>
-<li>LXQt-runner - Application launcher and calculator</li>
```

Comment:
This is not how to spell https://github.com/lxqt/lxqt-runner

Reply:
This was on purpose, for consistency with archiver ecc.

Reply:
Yes, and that is a reasonable critique of the naming-scheme used by the project, but it should be forwarded there.